README.md

```diff
@@ -58,15 +64,15 @@ If the cabinet file is more than 24 hours old or missing, this tool will automat
 
 Once the download is complete, it is checked against the SHA1 checksum present in the catalog.
 
-## Bash autocomplete
+## Bash Autocompletion
```

Comment:
Let's not change the style of any of the headings. This is intentional.

Reply:
By "style" I meant capitalization.

But the wording is also really fine as-is.